speech/snippets/change_speech_v2_location.py

```diff
@@ -11,63 +11,59 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-
-import argparse
+import os
 
 # [START speech_change_speech_v2_location]
 from google.api_core.client_options import ClientOptions
 from google.cloud.speech_v2 import SpeechClient
 from google.cloud.speech_v2.types import cloud_speech
 
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
```suggestion
PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
```

This line needs `import os` to be with in region tags

Reply:
Got it. I'll pull down `os` module into region tag in all samples👌🏻